packages/flutter/lib/src/animation/animation_controller.dart

```diff
@@ -704,13 +704,22 @@ class AnimationController extends Animation<double>
   /// provided, [duration] will be used instead, which has to be set before [repeat] is
   /// called either in the constructor or later by using the [duration] setter.
   ///
+  /// With [repeatTimes] set as valid value, animation will be executed by given times.
+  /// If value not given, infinite animation shall be simulated.
+  ///
   /// Returns a [TickerFuture] that never completes. The [TickerFuture.orCancel] future
   /// completes with an error when the animation is stopped (e.g. with [stop]).
```

Comment:
We should probably update this documentation, now that there's a way for the `TickerFuture` to complete.

```suggestion
  /// Returns a [TickerFuture] that never completes, unless a [repeatCount] is specified.
  /// The [TickerFuture.orCancel] future completes with an error when the animation is
  /// stopped (e.g. with [stop]).
```